docs/tutorial/population_dynamics.rst

```diff
@@ -157,6 +174,60 @@ the distribution values are directly available through the properties `Dwelltime
 `DwelltimeModel.bootstrap.lifetime_distributions` which return the data as a `numpy` array with
 shape `[# components, # bootstrap samples]`.
 
+.. _rate_constants:
+
+Assumptions and limitations on determining rate constants
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When using an exponential distribution to model biochemical kinetics, care must be taken to ensure that the model appropriately describes the
+observed system. Here we briefly describe the underlying assumptions and limitations for using this method.
+
+The exponential distribution describes the expected dwell times for states in a first order reaction where the rate of transitioning from
+the state is dependent on the concentration of a *single* component. A common example of this is the dissociation of a bound protein from
+a DNA strand:
+
+.. math::
+
+    \left[ \mathrm{DNA} \cdot \mathrm{protein} \right] \rightarrow \mathrm{DNA} + \mathrm{protein}
+
+This reaction is characterized by a rate constant :math:`k_\mathrm{off}` known as the dissociation rate constant with units of
+:math:`\mathrm{sec}^{-1}`.
+
+Second order reactions which are dependent on *two* reactants can also be determined in this way if certain conditions are met. Specifically,
+if the concentration of one reactant is much greater than that of the other, we can apply the *first order approximation*. This approximation
+assumes the concentration of the more abundant reactant remains approximately constant throughout the experiment and therefore does
+not contribute to the reaction rate. This condition is often met in single-molecule experiments; for example in a typical C-Trap experiment,
+the concentration of a protein in solution on the order of nM is significantly higher than the concentration of the single trapped tether.
+
+A common example of this is the binding of a protein to a DNA strand:
+
+.. math::
+
+    \mathrm{DNA} + \mathrm{protein} \rightarrow \left[ \mathrm{DNA} \cdot \mathrm{protein} \right]
+
+This reaction is described by the second order association rate constant :math:`k_\mathrm{on}` with units of :math:`\mathrm{M}^{-1}\mathrm{sec}^{-1}`.
+Under the first order approximation, this can be determined by fitting the appropriate dwell times to the exponential model and multiplying
+the resulting rate constant by the concentration of the protein in solution.
```

Comment:
So, what is 'this' in the sentence, 'this can be determined'? 
The rate that you get out of the dwell time distribution is effectively kon * [Protein], right? So you would have to divide the fitted rate constant by concentration if you want to obtain kon? Do I think about it wrongly, or were you trying to say something else?

Reply:
And I see you mention '_appropriate_ dwell times'. I guess the word 'appropriate' is hiding a whole discussion on itself :P

Reply:
no, you're right. I was apparently just not thinking when I wrote this